### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by zhvirblis**

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function getStringLength(value) {
  *   'Chuck','Norris'  => 'Hello, Chuck Norris!'
  */
 function getStringFromTemplate(firstName, lastName) {
-    throw new Error('Not implemented');
+    return 'Hello, ' + firstName + ' ' + lastName+ '!';
 }
```
Comment (IlyaMokin) [inline task/01-strings-tasks.js]: see link above the method

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -238,7 +257,10 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    if(typeof value=='object'){
+        return Object.prototype.toString.call(value)=='[object String]';
+    }
+    return (typeof value)=='string';
 }
```
Comment (IlyaMokin) [inline task/01-strings-tasks.js]: see typeof and instanceof operators

File: task/01-strings-tasks.js [hunk 3]
```diff
@@ -201,7 +201,21 @@ function extractEmails(str) {
...
+            else if(i==0 && j==width-1) str+='┐\n';
+            else if(j!=0 && i==height-1) str+='─';
+            else if(j==0 && i==height-1) str+='└';
+            else if(i!=0 && j==0) str+='│';
+            else if(i!=0 && j==width-1) str+='│\n';
+            else str+=' ';
+        }
+    }
+    return str;
 }
```
Comment (AlexMokin) [inline task/01-strings-tasks.js]: use repeat function. Do it without for loops

File: task/01-strings-tasks.js [hunk 4]
```diff
@@ -267,7 +286,25 @@ function isString(value) {
...
+    else{
+        index+=Number(value[0])-1;
+    }
+    if(value[1]=='0'){
+        index=symbols[value[2]]*SYMBOL_STEP+FOR_TEN;
+    }
+    else{
+        index+=symbols[value[1]]*SYMBOL_STEP;
+    }
+    return index;
```
Comment (AlexMokin) [inline task/01-strings-tasks.js]: do it without if and switch statements. If takes too long, take a look at nastya's solution

**Comment by IlyaMokin** [comment]
done

**Pull request closed without merging**